dune-project

```diff
@@ -84,7 +84,11 @@ structured way."))
 (package
  (name coqide)
  (depends
+  (ocamlfind :build)
+  (conf-findutils :build)
+  (conf-adwaita-icon-theme :build)
```

Comment:
Why is it `build` here but not in the .opam?

Reply:
This may be due to some limitation, but to my knowledge `conf-adwaita-icon-theme` is a regular dep, not just a build dep.

Reply:
A mistake from my side. However, Dune forces me to give an argument to the package (either `:build` or a version constraint). So I will have to use `(>= 1)` instead here.

Reply:
Does it work if you drop the parenthesis?

Reply:
[It does]

Reply:
Is this the change you are testing?

```diff
diff --git a/dune-project b/dune-project
index 42724f000a..4f277c7af7 100644
--- a/dune-project
+++ b/dune-project
@@ -85,7 +85,7 @@ structured way."))
  (depends
   (ocamlfind :build)
   (conf-findutils :build)
-  (conf-adwaita-icon-theme (>= 1))
+  (conf-adwaita-icon-theme)
   (coqide-server (= :version))
   (cairo2 (>= 0.6.4))
   (lablgtk3-sourceview3 (>= 3.1.2)))
```

I get:

```
dune build  --root . theories_dune ltac2_dune
File "dune-project", line 88, characters 27-27:
88 |   (conf-adwaita-icon-theme)
                                
Error: Premature end of list
make: *** [Makefile:124: .dune-stamp] Error 1
```

with Dune 3.6.1.